modules/system/assets/js/snowboard/main/Snowboard.js

```diff
@@ -336,7 +336,6 @@ export default class Snowboard {
         const listeners = this.listensToEvent(eventName);
         if (listeners.length === 0) {
             this.debug(`No listeners found for global event "${eventName}"`);
-            return true;
```

Comment:
Is this intentional?

Reply:
Yes, it short-circuits the event listeners if there are none available.